onsset/runner.py

```diff
@@ -333,15 +333,14 @@ def scenario(specs_path, calibrated_csv_path, results_folder, summary_folder, pv
                 'full_life_cycles': 2000  # Equivalent full life-cycles of battery until replacement
             }
 
-            mg_diesel_calc = Technology(om_of_td_lines=0.02,
-                                        distribution_losses=0.05,
-                                        connection_cost_per_hh=100,
-                                        base_to_peak_load_ratio=0.85,
-                                        capacity_factor=0.7,
+            #Used to model LCOE for not-served-energy
+            mg_diesel_calc = Technology(base_to_peak_load_ratio=0.85,  # Deducted from curves
+                                        capacity_factor=0.7, #Typical for diesel generators
                                         tech_life=15,
                                         om_costs=0.1,
```

Comment:
I think the specs should use the _sa_diesel_calc_ instead, as here we assume each consumer has their own back-up generator? (I.e. cost per kW and such would be higher than for a mini-grid scale generator)